### PR TITLE
Added update to run the task

### PR DESCRIPTION
**Pull request by joinimran**
Updated the document as suggested by the user to update the XML file created by GP preference. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4797

### CODE REVIEW

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-gp.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ ms.date: 04/24/2018
 > [!NOTE]
 > To use Group Policy (GP) updates to deploy the package, you must be on Windows Server 2008 R2 or later.
 
+> For Windows Server 2019, you may need to replace NT AUTHORITY\Well-Known-System-Account with NT AUTHORITY\SYSTEM of XML file that Group Policy preference creates.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-gp.md]:
```suggestion
> For Windows Server 2019, you may need to replace NT AUTHORITY\Well-Known-System-Account with NT AUTHORITY\SYSTEM of the XML file that the Group Policy preference creates.
```

**Comment by e0i** [comment]
@joinimran 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by mypil** [comment]
@mjcaparas - PR has been copy edited and is ready for final review. This is based on issue #4797.

Thank you.

**Pull request merged**